pdata/pmetric/metrics.go

```diff
@@ -102,19 +102,22 @@ func (ms Metrics) DataPointCount() (dataPointCount int) {
 type MetricType int32
 
 const (
-	MetricTypeNone MetricType = iota
+	MetricTypeEmpty MetricType = iota
```

Comment:
Add a comment to the value that it means none of the "possible" values is set? Or something in that lines.

Reply:
Added a comment to these